### ISSUE TITLE
Generated Client Code Compile Failed!

### ISSUE THREAD

**Issue by wangzhizhou** [apple/swift-openapi-generator#83]
I use the Hangar API  OpenAPI Document of PaperMC to generate client， build when I build, there occur some error.

The OpenAPI Document is:  Hangar API: https://hangar.papermc.io/v3/api-docs.yaml

**Comment by czechboy0** [comment]
Hi @wangzhizhou, thanks for filing the issue.

The problem you're hitting is that Swift OpenAPI Generator doesn't yet support query parameters that are object schemas. We're tracking this feature request in #35.

Until the issue is fixed, as a temporary workaround, you could comment out the `pagination` query parameter, which will make it not get generated, and hopefully that can unblock you for the time being. You could use something like a `ClientMiddleware` to inject the query parameter, if you really need to send it.

Since this is a duplicate, I'll close this issue, but it'll remain linked to the original #35, so please subscribe to get updates on that issue. Thanks!

**Comment by wangzhizhou** [comment]
ok, I will follow the #35

**Comment by wangzhizhou** [comment]
is this problem caused by OpenAPIKit ? or just because swift-openapi-generator unimplentation this feature of openapi spec ?

**Comment by wangzhizhou** [comment]
https://github.com/mattpolzin/OpenAPIKit/issues/288

**Comment by czechboy0** [comment]
No, this isn't an issue with OpenAPIKit, it's an unimplemented feature in Swift OpenAPI Generator.
